src/pages/Profile.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import ProfileSection from '../components/ProfileSection';
+// import MissionProfile from '../components/MissionProfile';
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused codes as comments make your code not clean.